python/pyspark/sql/tests/typing/test_session.yml

```diff
@@ -51,25 +51,6 @@
     spark.createDataFrame(["foo", "bar"], "string")
 
 
-- case: createDataFrameScalarsInvalid
```

Comment:
Because of PyArrow's lack of type annotations, the changes in this PR caused this typing test to fail.

However, the failure modes checked in this test are adequately caught without relying on PySpark's type annotations.

For the first case, PySpark raises:
```
pyspark.errors.exceptions.base.PySparkTypeError: [CANNOT_INFER_SCHEMA_FOR_TYPE] Can not infer schema for type: `str`.
```

For the second case, PySpark raises:
```
pyspark.errors.exceptions.base.PySparkTypeError: [FIELD_DATA_TYPE_UNACCEPTABLE] StructType([StructField('name', StringType(), True), StructField('age', IntegerType(), True)]) can not accept object 1 in type <class 'int'>.
```

Given that the errors are caught by other means, it seems reasonable to remove this test.

Reply:
I added tests in 40072d0 to ensure we still cover these error conditions somewhere in the tests.